docs/build-with-powerloom/devnet/FAQs.md

```diff
@@ -0,0 +1,95 @@
+
+**Note:** Please do not use your NFT minting wallet to run the Snapshotter lite node. The safest approach is to generate and assign a burner wallet using tools like [Vanity-ETH](https://vanity-eth.tk/) and [Powerloom Burner Wallet Generator](https://snapshotter-dashboard.powerloom.network/burner)
+
+### What security measures should I consider when creating a burner wallet?
+When creating a burner wallet, prioritize security by managing private keys securely, limiting funds stored, performing offline transactions, regularly backing up data, ensuring a secure environment, using trusted sources, considering multi-signature wallets, and staying informed about security practices.
+    
+### How do I deploy my Snapshotter node once the environment is configured?
+After configuring the environment variables, set the powerloom_deploy codebase by running the `bootstrap.sh` command in the terminal. Developers may use different shells, such as bash or zsh.
+
+**Note**: This is a one-time step that resets the `powerloom_deploy` codebase to the latest version of the `devnet` branch. After completing this one-time setup, you'll be ready to dive into the codebase and start building amazing data applications!
+    
+### What chains are supported?
+We support Ethereum Mainnet and Sepolia Testnet.
+    
+### What steps should I follow to perform a clean reset of my node?
+You can do a clean **reset** by running the following command before restarting.
+```markdown
+    docker-compose --profile ipfs down --volumes
+```
+or
+```markdown
+    docker compose --profile ipfs down --volumes
+```
+    
+### How can I stop the node?
+To shut down services, use the keyboard shortcut **`Ctrl + C`** on macOS/Linux or **`Ctrl + Break`** on Windows. Run the provided Docker command before restarting the node setup for a clean reset.
+    
+### Can we run the setup on different languages?
+Yes, you can run the setup in different languages. The Snapshotter node is a complex system with multiple components written in different languages. Pooler is a Python program, while Audit Protocol is coded in Go. Since Pooler and Audit Protocol communicate via RabbitMQ, either component can be ported to different languages, making the Snapshotter node accessible to developers unfamiliar with Python or Go.
+    
+### What is the purpose of the PROTOCOL_STATE_CONTRACT variable?
+This variable specifies the protocol state's contract address, which depends on the Epoch size. Currently, there are options for Epoch sizes of 1 and 10, allowing developers to experiment with different use cases.
+
+### What is the Relayer Host?
+The Relayer Host is the relayer URL for the Powerloom Protocol Chain. The relayer is a node within a decentralized network responsible for relaying or forwarding messages, transactions, or data between different participants. Relayers play a crucial role in facilitating communication and interoperability. 
```

Comment:
too generic. This should state that Powerloom presently operates relayers that take care of transactions that submit snapshots to the protocol state contract on behalf of the participating snapshotter nodes. This ensures they do not have to spend gas fees to submit snapshots on the Powerloom protocol chain, aka Prost.

Reply:
The Relayer Host serves a specific function within the Powerloom Protocol:

- Functionality: It is the URL for the relayer, a specialized node within the decentralized network.
- Role of Relayers: These nodes are responsible for processing transactions that involve submitting snapshots to the protocol state contract.
- Cost Efficiency: By handling these transactions, relayers ensure that participating snapshotter nodes do not have to pay gas fees themselves.
- Benefits: This setup allows snapshotter nodes to operate more efficiently on the Powerloom protocol chain, known as Prost, by reducing their operational costs.

This arrangement is crucial for maintaining smooth and cost-effective operations within the Powerloom ecosystem.